tests/acceptance/vote-page/course-extension-ideas-test.js

```diff
@@ -25,6 +25,7 @@ module('Acceptance | vote-page | course-extension-ideas', function (hooks) {
     assert.strictEqual(votePage.selectedCourseName, 'Build your own Redis');
     assert.strictEqual(votePage.courseExtensionIdeaCards.length, 2);
     assert.strictEqual(votePage.findCourseExtensionIdeaCard(courseExtensionIdea.name).voteButtonText, '1 vote');
+    assert.true(votePage.findCourseExtensionIdeaCard(courseExtensionIdea.name).isGreyedOut);
```

Comment:
It's not super clear why this should be grayed out + we aren't testing whether other ideas are not grayed out. A more easy-to-read test would be something like this: 

```
const releasedIdeaCard = votePage.findCourseExtensionIdeaCard(...)
const notStartedIdeaCard = votePage.findCourseExtensionIdeaCard(...)

assert.strictEqual(releasedIdeaCard.developmentStatusLabelText, "released")
assert.true(releasedIdeaCard.isGreyedOut)

assert.false(releasedIdeaCard.developmentStatusLabel.isPresent)
assert.false(releasedIdeaCard.isGreyedOut)
```

This mimics what a human would look for - are released ideas grayed out and marked as so? And are the other ones not greyed out?

Reply:
makes sense